scripts/breaking_changes_checker/detect_breaking_changes.py

```diff
@@ -395,7 +396,7 @@ def main(package_name: str, target_module: str, version: str, in_venv: Union[boo
     pkg_dir = os.path.abspath(args.target_package)
     package_name = os.path.basename(pkg_dir)
     logging.basicConfig(level=logging.INFO)
-    if package_name not in RUN_BREAKING_CHANGES_PACKAGES:
+    if package_name not in RUN_BREAKING_CHANGES_PACKAGES and not any(bool(re.findall(p, package_name)) for p in RUN_BREAKING_CHANGES_PACKAGES):
```

Comment:
It is more convenient to support regex when allow detection for all Mgmt SDK.

Reply:
We might have to add tests for this eventually, what if there's a mgmt library we do want to skip checks on?

Reply:
We could optimize the logic in the future. At least for now, it is very convenient.